src/i18n/ru/print.js

```diff
@@ -0,0 +1,35 @@
+export default {
+  keyboard: {
+    label: 'Клавиатура'
+  },
+  layout: {
+    label: 'Раскладка'
+  },
+  layer: {
+    label: 'Слой'
+  },
+  anonymous: {
+    label: 'Аноним'
```

Comment:
Sorry, I can't find where this field is used for better translation. Can you please provide a screenshot or at least tell me what part of speech this word is supposed to be?

Reply:
Yeah, it's if you don't supply a field during the layer print out screen.

Reply:
@yanfali Got it, thank you!